### PR TITLE
Add app Icon

### PR DESCRIPTION
**Pull request by NullRa**

### CODE REVIEW

**Comment by ender503** [comment]
Could we give the icon image name consistently? 🤔  e.g. Icon-{size}@{scale}.png

**Comment by NullRa** [comment]
Do you mean like this?
Rename "Icon-60.png" to "Icon-60*60.png"
Rename "Icon-60@2x.png" to "Icon-120*120@2x.png"

File: Transportation_PLUS/Assets.xcassets/AppIcon.appiconset/Contents.json [hunk 1]
```diff
@@ -86,8 +112,9 @@
       "scale" : "2x"
     },
     {
-      "idiom" : "ios-marketing",
       "size" : "1024x1024",
+      "idiom" : "ios-marketing",
+      "filename" : "icon1024_1024.png",
```
Comment (ender503) [inline Transportation_PLUS/Assets.xcassets/AppIcon.appiconset/Contents.json]: I mean rename the `icon1024_1024` to `Icon-1024.png`. 😏

File: Transportation_PLUS/Assets.xcassets/AppIcon.appiconset/Contents.json [hunk 2]
```diff
@@ -1,43 +1,69 @@
...
+      "size" : "57x57",
       "idiom" : "iphone",
-      "size" : "60x60",
+      "filename" : "Icon.png",
+      "scale" : "1x"
+    },
+    {
+      "size" : "57x57",
+      "idiom" : "iphone",
+      "filename" : "Icon@2x.png",
```
Comment (ender503) [inline Transportation_PLUS/Assets.xcassets/AppIcon.appiconset/Contents.json]: And `Icon-57@2x.png`.

File: Transportation_PLUS/Assets.xcassets/AppIcon.appiconset/Contents.json [hunk 3]
```diff
@@ -1,43 +1,69 @@
...
       "size" : "40x40",
+      "idiom" : "iphone",
+      "filename" : "Icon-Spotlight-40@3x.png",
       "scale" : "3x"
     },
     {
+      "size" : "57x57",
       "idiom" : "iphone",
-      "size" : "60x60",
+      "filename" : "Icon.png",
```
Comment (ender503) [inline Transportation_PLUS/Assets.xcassets/AppIcon.appiconset/Contents.json]: And `Icon-57.png`.

File: Transportation_PLUS/Assets.xcassets/AppIcon.appiconset/Contents.json [hunk 4]
```diff
@@ -1,43 +1,69 @@
...
       "size" : "20x20",
+      "idiom" : "iphone",
+      "filename" : "Icon-Spotlight-41.png",
       "scale" : "2x"
     },
     {
-      "idiom" : "iphone",
       "size" : "20x20",
+      "idiom" : "iphone",
+      "filename" : "Icon-61.png",
```
Comment (ender503) [inline Transportation_PLUS/Assets.xcassets/AppIcon.appiconset/Contents.json]: And `Icon-20@3x.png`.

File: Transportation_PLUS/Assets.xcassets/AppIcon.appiconset/Contents.json [hunk 5]
```diff
@@ -1,43 +1,69 @@
 {
   "images" : [
     {
-      "idiom" : "iphone",
       "size" : "20x20",
+      "idiom" : "iphone",
+      "filename" : "Icon-Spotlight-41.png",
```
Comment (ender503) [inline Transportation_PLUS/Assets.xcassets/AppIcon.appiconset/Contents.json]: And `Icon-20@2x.png`

**Comment by ender503** [comment]
It would be great to make the images that have a consistent name. 😏

File: Transportation_PLUS/Assets.xcassets/AppIcon.appiconset/Contents.json (hunk 1 above)
Comment (NullRa) [reply Transportation_PLUS/Assets.xcassets/AppIcon.appiconset/Contents.json]: Oh! I got it!

**Comment by NullRa** [comment]
OK, I finished. Thanks!

File: Transportation_PLUS/Assets.xcassets/AppIcon.appiconset/Contents.json [hunk 6]
```diff
@@ -86,8 +112,9 @@
       "scale" : "2x"
     },
     {
-      "idiom" : "ios-marketing",
       "size" : "1024x1024",
+      "idiom" : "ios-marketing",
+      "filename" : "icon-1024@1x.png",
```
Comment (ender503) [inline Transportation_PLUS/Assets.xcassets/AppIcon.appiconset/Contents.json]: Rename to `Icon-1024@1x.png` just like its friends(start with uppercase).  😏

Comment (NullRa) [reply Transportation_PLUS/Assets.xcassets/AppIcon.appiconset/Contents.json]: Oh! thx!

**Pull request merged**